### DISCUSSION TITLE
Spellcheck actions  source control panel (commit message box)

### DISCUSSION THREAD

**Discussion by ciarantr** [streetsidesoftware/vscode-spell-checker#2377] [Q&A]
Is it possible to use spellcheck actions in the source control panel? The errors are displays however when I try select the text I only get the option to cut, copy, paste.

**Comment by Jason3S** [comment]
This seems to be a VS Code limitation. When VS Code first enabled spell checking the source control panel, it allowed running the actions, but it caused an error in VS Code. I think they chose to disable the actions until they could fix the underlying issue.

VS Code Tracking issue: https://github.com/microsoft/vscode/issues/155282
Original Issue: https://github.com/microsoft/vscode/issues/35571#issuecomment-1188268657

<img width="537" alt="image" src="https://user-images.githubusercontent.com/3740137/204881637-97879e32-e73f-4b7f-8e27-327e18ff5557.png">
Notice that `Quick Fix` hasn't been implemented yet.

**Reply by ciarantr** [reply]
Thanks for getting back to me so quickly Jason. I wasn’t sure, I thought at one stage I could use the quick actions to correct spelling mistakes but I was wrong.

**Comment by mjmatthiesen** [comment]
Hi,

Any updates on this by chance? Even just seeing something is spelled wrong is enough, I don't need the actions.

Thanks!

**Reply by Jason3S** [reply]
@mjmatthiesen,

This had been working. I'm not sure why it might have stopped.

**Reply by mjmatthiesen** [reply]
Is there any specific config I need maybe? I'm not getting anything underlined in the git commit message.

**Reply by Jason3S** [reply]
@mjmatthiesen,

I'm not either. I'll investigate.

**Reply by Jason3S** [reply]
Spell checking the commit message does not work on Windows, but does work on MacOS. Need to find out why.
